src/software/jetson_nano/thunderloop.cpp

```diff
@@ -416,6 +417,29 @@ double Thunderloop::getCpuTemperature()
     }
 }
 
+static bool isPowerStable()
```

Comment:
you should define this function as:
`isPowerStable(std::string log_file)`.

Then, add some unit tests for this function with the GoogleTest suite using mocked log files